lib/tools/xctest-commands.js

```diff
@@ -1,20 +1,31 @@
-import { fixOutputToArray } from '../helpers';
+import { fixOutputToArray, convertToIDBEnv } from '../helpers';
 
 const xctestCommands = {};
 
+/**
+ * @typedef {Object} RunOptions
+ *
+ * @property {string} env Environment variables to be passed to the test runner
```

Comment:
perhaps Object? Also, it would be good to describe the format of this object items